### PR TITLE
feature/mse-tab

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

MSE Tab for node
## Impacts

Admin GUI Node View
## NEWS file entries

New Features
++++++++++++
- Able to view Cisco MSE information
## Delete branch after merge

NO

### CODE REVIEW

File: conf/documentation.conf [hunk 1]
```diff
@@ -1901,3 +1901,28 @@ options=enabled|disabled
...
+[mse_tab.server]
+type=text
+description=<<EOT
+Server name for MSE service
+EOT
+
+[mse_tab.user]
+type=text
+description=<<EOT
+user name for MSE service
```
Comment (extrafu) [inline conf/documentation.conf]: Username

File: conf/documentation.conf [hunk 2]
```diff
@@ -1901,3 +1901,28 @@ options=enabled|disabled
...
+EOT
+
+[mse_tab.pass]
+type=text
+description=<<EOT
+password for MSE service
```
Comment (extrafu) [inline conf/documentation.conf]: Password

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 3]
```diff
@@ -8929,6 +9020,10 @@ msgstr "Show parking portal"
 msgid "parking.threshold"
 msgstr "Parking threshold"
 
+# conf/documentation.conf (mse_tab.pass)
+msgid "password for MSE service"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: Password

File: html/pfappserver/lib/pfappserver/I18N/en.po [hunk 4]
```diff
@@ -9850,6 +9950,10 @@ msgstr "Upstream URL"
 msgid "upstream.submit_url"
 msgstr "Upstream Submit URL"
 
+# conf/documentation.conf (mse_tab.user)
+msgid "user name for MSE service"
```
Comment (extrafu) [inline html/pfappserver/lib/pfappserver/I18N/en.po]: Username

File: html/pfappserver/lib/pfappserver/Model/Node/Tab/MSE.pm [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+=head2 process_tab
+
+Process Tab
+
+=cut
+
+sub process_tab {
+    my ($self, $c, @args) = @_;
+    my $rest = Net::Cisco::MSE::REST->new(%{$Config{mse}});
```
Comment (fdurand) [inline html/pfappserver/lib/pfappserver/Model/Node/Tab/MSE.pm]: mse_tab ?

File: conf/documentation.conf [hunk 6]
```diff
@@ -1901,3 +1901,28 @@ options=enabled|disabled
...
 EOT
+
+[mse_tab.enabled]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Enable MSE
+EOT
+
+[mse_tab.server]
```
Comment (fdurand) [inline conf/documentation.conf]: mse_tab.url like http://localhost:8083/

**Comment by jrouzierinverse** [comment]
re-review

**Comment by louismunro** [comment]
Enough review.
Merged for testing.

**Pull request merged**